game.pro

```diff
@@ -33,10 +33,10 @@ unix:!macx {
 }
 
 win32{
-  INCLUDEPATH += C:/Qt/sfml/include
-  INCLUDEPATH += D:/Qt/sfml/include
   LIBS += -LC:/Qt/sfml/lib
   LIBS += -LD:/Qt/sfml/lib
+  LIBS += -LC:/DEV/SFML-2.5.1/lib
+  INCLUDEPATH += /home/pohaha/Downloads/SFML-2.5.1/include

```

Comment:
This INCLUDEPATH clearly adds a Linux path, while being in the win32 section. I hope you will remove it, else I will create an Issue for a beginner to do so :-)